### ISSUE TITLE
NaN values in TensorBoard for all losses

### ISSUE THREAD

**Issue by tr1pzz** [NVlabs/stylegan2-ada#7]
Is anyone else getting NaN values in Tensorboard for all of the losses? (Training runs fine, reporting doesn't)
![image](https://user-images.githubusercontent.com/24993604/95960256-ceabfa00-0e03-11eb-8814-643923343b1e.png)

**Comment by VladimirGl** [comment]
Same for me. From source code it should assert if this values are None in `report_loss` function in `training/loss.py`.

**Comment by jzhu1234** [comment]
Same here as well. @tr1pzz does your scores at least seem reasonable?

**Comment by tkarras** [comment]
Thank you for reporting this. I've checked in a bugfix.

**Comment by tr1pzz** [comment]
Adding your bugfix actually gives me an error (Running on a single RTX 2080 Ti with Cuda 10.2):

`F tensorflow/core/kernels/cuda_solvers.cc:94] Check failed: cusolverDnCreate(&cusolver_dn_handle) == CUSOLVER_STATUS_SUCCESS Failed to create cuSolverDN instance.`

This error is triggered on line 247 of training_loop.py: (it's the G_train_op that causes it to trigger)
`tflib.run([G_train_op, data_fetch_op])`
